src/main/java/address/controller/MainController.java

```diff
@@ -185,7 +183,7 @@ private void showFooterStatusBar() {
             GridPane gridPane = loader.load();
             gridPane.getStyleClass().add("grid-pane");
             StatusBarFooterController controller = loader.getController();
-            controller.initStatusBar(config);
+            controller.initStatusBar(config, modelManager.getPrefs());
```

Comment:
Should we name the method as `setConnections`? To be consistent with other controllers. If there is other initialization needed, maybe have `start` after `setConnections` or put in constructor.